llvm/include/llvm/ADT/FunctionExtras.h

```diff
@@ -152,7 +152,7 @@ template <typename ReturnT, typename... ParamTs> class UniqueFunctionBase {
       void *StoragePtr;
       size_t Size;
       size_t Alignment;
-    } OutOfLineStorage;
+    } OutOfLineStorage = {};
```

Comment:
note: i'm still not sure what will this do (hopefully, zero-initialize) but whatever. if it silences the things, sure.

I would've done this though:
```cpp
    struct OutOfLineStorageT {
      void *StoragePtr = nullptr;
      size_t Size = 0;
      size_t Alignment = 0;
    } OutOfLineStorage = {};
```

Reply:
as void* and size_t are primitive types, {} should initialize them with zero, yes